.github/workflows/test.yml

```diff
@@ -20,14 +20,6 @@ on:
       - ".codecov.yml"
       - Sentry.xcodeproj
 
-      # run the workflow any time an Xcode scheme changes for a sample app
-      - "Samples/tvOS-Swift/tvOS-Swift.xcodeproj/xcshareddata/xcschemes/tvOS-Swift.xcscheme"
-      - "Samples/iOS-Swift/iOS-Swift.xcodeproj/xcshareddata/xcschemes/iOS13-Swift.xcscheme"
-      - "Samples/iOS-Swift/iOS-Swift.xcodeproj/xcshareddata/xcschemes/iOS-SwiftUITests.xcscheme"
-      - "Samples/iOS-Swift/iOS-Swift.xcodeproj/xcshareddata/xcschemes/iOS-Swift.xcscheme"
```

Comment:
I believe we should keep at least iOS-Swift, otherwise we fix a problem in the app itself and the UI test wont run.

Reply:
I see. Perfect